scipy/linalg/_solvers.py

```diff
@@ -329,44 +343,81 @@ def solve_discrete_are(a, b, q, r):
     Notes
     -----
     Method taken from:
-    Laub, "A Schur Method for Solving Algebraic Riccati Equations."
-    U.S. Energy Research and Development Agency under contract
-    ERDA-E(49-18)-2087.
-    http://dspace.mit.edu/bitstream/handle/1721.1/1301/R-0859-05666488.pdf
+    P. van Dooren , "A Generalized Eigenvalue Approach For Solving Riccati 
```

Comment:
I know it was like this in the original, but it would be better to put this in a references section and just cite it here. See:

https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#docstring-standard